### DISCUSSION TITLE
Shortcut for selecting/switching between results tabs

### DISCUSSION THREAD

**Discussion by acsel87** [dbeaver/dbeaver#13525] [Ideas]
Hi
I was searching for a solution, regarding the tooltip when hovering results tab, described in #9472

Instead of an option to disable the tooltip (on mouse hover), I was thinking of adding one or both of the following shortcuts for selecting result tabs:

_[ I did not find any similar shortcuts in the editor ]_

1. key + [number] : selects the tab specified by the number (as CTRL + [number] does for tabs in chrome)
2. key : select last selected tab, which would be especially helpful for comparing two results as it would allow you to switch between them without worrying about what the number/order of the tabs

Personally, for this specific usage (comparing two tabs) I find the second option way more useful.

Thank you

**Comment by qupear** [comment]
As a workaround I press TAB to focus results and CTRL+PGUP/PGDOWN then to switch sets. But CTRL+0..9 would be very convenient.
